Misc/NEWS.d/next/Library/2022-09-14-19-15-01.gh-issue-87079.0zYmW5.rst

```diff
@@ -0,0 +1,2 @@
+Warn the user whenever asyncio event loops override a signal wake up file
+descriptor that was previously set.
```

Comment:
```suggestion
:mod:`asyncio` now emits :exc:`ResourceWarning` when overriding signal wakeup fd if it was previously set.
```

Reply:
Well dang I missed this comment and merged without applying that. Sorry about that (I fail at scrolling). I think this isn't so important that we need to send another PR, but if you insist I'll gladly merge it.

Reply:
No worries, it was just a minor suggestion.